ngraph/frontend/onnx_editor/include/onnx_editor/edge_mapper.hpp

```diff
@@ -26,7 +26,7 @@ namespace ngraph
         class EdgeMapper
         {
         public:
-            EdgeMapper() = default;
+            ONNX_IMPORTER_API EdgeMapper() = default;
```

Comment:
Why exporting per method is used instead of standard 'class ONNX_IMPORTER_API EdgeMapper' (this way is used in other places of ngraph-core)?

Reply:
I don't want to export ctor with protobufs entities